Gemfile

```diff
@@ -22,6 +22,7 @@ gem 'phantomjs'
 gem 'textacular', '~> 5.0'
 gem 'carrierwave', '~> 1.0'
 gem 'carrierwave-aws'
+gem 'carrierwave-i18n'
```

Comment:
Nice digging on this!!!
I looked at this carrier wave issue:  https://github.com/carrierwaveuploader/carrierwave/issues/2173
I actually added the gem around the time this issue was created and haven't updated it since.  It seemed like carrier wave had resolved it, so I ran 'bundle update carrierwave' in my terminal locally, which bumped the version up to 1.2.1 and fixed the problem.  So, we should remove the carrierwave-i184 gem, run bundle update on carrier wave, and commit the Gemfile.lock changes.

Reply:
@benhayehudi this is the second of two outstanding changes.  See comment above.  Let me know if you have any questions!